vignettes/ORFikOverview.Rmd

```diff
@@ -148,6 +149,23 @@ The uORFs from this example will most likely contain some uORFs not in the origi
 example, also some will be lost. This is, because different tissues transcribes
 different mRNAs.
 
+# Syntax of Orfs in ORFik
+
+The ORFs in ORFik are grouped in a GRangesList. Each group is the exons per orf.
+The names of the GRangesList is the orf names, named as transcriptName_*, 
+where transcriptName is the transcriptName i.g for ensembl: "ENST123101239".
+The "_" is the seperator
```

Comment:
We will have to change this "naming" convention in the future to use outside vs inside mcols.